AlphaWallet/Activities/Coordinators/ActivitiesCoordinator.swift

```diff
@@ -45,6 +47,7 @@ class ActivitiesCoordinator: NSObject, Coordinator {
         let viewModel = ActivitiesViewModel()
         let controller = ActivitiesViewController(viewModel: viewModel, sessions: sessions)
         controller.delegate = self
+        controller.tabBarItem = UITabBarController.Tabs.activities.tabBarItem
```

Comment:
But assigning to `.tabBarItem` doesn't seem like it should be the individual coordinators' (or view controllers) responsibility, it should be higher up? `InCoordinator` (where it used to be in) or something that `InCoordinator` delegates to for tab configuration/management seems more appropriate?